src/bregman.jl

```diff
@@ -7,14 +7,17 @@ mutable struct BregmanParams
     maxIter
     store_trace
     antichatter
-    quantile
     alpha
     spg
+    TD
+    quantile
+    λ
+    λfunc
 end
 
 """
     bregman_options(;verbose=1, optTol=1e-6, progTol=1e-8, maxIter=20
-                    store_trace=false, linesearch=false, alpha=.25, spg=false)
+                    store_trace=false, λ=.2, alpha=.25, spg=false)
```

Comment:
where does .2 comes from

Reply:
just to inform user that you can do something like this